src/engine/sys/sdl_glimp.cpp

```diff
@@ -962,67 +962,55 @@ GLimp_InitExtensions
 ===============
 */
 
-static void RequireExt( bool hasExt, const char* name )
-{
-	if ( hasExt )
-	{
-		logger.WithoutSuppression().Notice("...using GL_%s", name );
-	}
-	else
-	{
-		Sys::Error("Required extension GL_%s is missing", name );
-	}
-}
+#define NoticeExtUnsupported(name) logger.WithoutSuppression().Notice("...GL_%s unsupported", #name)
 
-static bool LoadExtWithCvar( bool hasExt, const char* name, bool cvarValue )
```

Comment:
What does `isCore` mean?

Reply:
The code does not test for the explicit extension if the driver advertises OpenGL Core 3 and we know Core 3 requires this extension.

I don't know which buggy driver would advertise GL Core 3 but not the related extensions, but that was the previous behavior I just refactored: when using the right macro, core extensions are checked by their name only if core is not there, typically the case with GL `Core 2.1 + ARB_framebuffer_object`, the lowest configuration we support.